### PR TITLE
MOTECH-767 Documenting the Commcare module

### PR DESCRIPTION
**Pull request by craigappl**

### CODE REVIEW

File: docs/source/commcare.rst [hunk 1]
```diff
@@ -1,7 +1,122 @@
...
 ===============
-CommCare Module
+Commcare Module
 ===============
 
-To be written.
+.. contents::
+   :depth: 3
+
+The Commcare module allows MOTECH to interact with `Commcare applications <https://www.commcarehq.org>`_ that are hosted anywhere in the world. MOTECH has the capability to receive forwarded forms and cases as well as pull historical forms and cases from existing Commcare applications. These inputs can be used in the tasks module as task triggers that can perform many more task actions including storing information in MOTECH, forwarding data to DHIS2 and contact the client. Note that MOTECH offers point-to-point interaction with the Commcare API which requires a `paid Commcare plan <https://www.commcarehq.org/software_services/#software-services-pricing>`_.
```
Comment (pgesek) [inline docs/source/commcare.rst]: contact the client -> contacting the client

File: docs/source/commcare.rst [hunk 2]
```diff
@@ -1,7 +1,122 @@
...
+
+There are a few Commcare terms that are useful to know as you proceed through the documentation.  A project is the unit where you are able to assign the workforce and . You are also billed at the project level. Each project can have multiple applications which are independent units within a project. For example, you may have a Maternal and Child Health project that has a maternal application and a child immunization application. Each application is independent. Within each application, you are able to create modules, or collections of forms which are displayed to the end user who enter the data from the field. These forms are able to contribute to a case, which allows you to track changes to something over time. To continue the example, the maternal application may have a prenatal care module that has a registration form, a 3 month, 6 month and 9 month checkup form and a delivery form. Each of these forms would contribute to the case, allowing you to follow a mother over time until she delivers.
```
Comment (pgesek) [inline docs/source/commcare.rst]: workforce and . The end of the statement is missing

**Comment by craigappl** [comment]
@pgesek I see the code review -1 label from 8 days ago, but can't find any comments.

**Comment by pgesek** [comment]
@craigappl not sure what I was doing, I think I removed the label but then though better of it and asked you on email whether this is ready. Anyway, I will take a look.

**Pull request merged**